connectivity/manifests/client-egress-to-cidr-deny.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  namespace: cilium-test
+  name: client-egress-to-cidr-deny
+spec:
+  endpointSelector:
+    matchLabels:
+      kind: client
+  egressDeny:
```

Comment:
I don't now remember the semantics a policy with a sole deny rule should have... Isn't this denying everything, like if there was no rules in the policy? Should we add some allow rules to be able to make a distinction between deny/allow? Especially if a wider rule allows (e.g., everything to a entity), and a deny rule then denies some ports (that would be allowed based on the 1st rule)??

Reply:
this is good point, I didn't know about default deny till recently.

Each deny tests now are having one extra policy (allow all except world) and one positive test.